### ISSUE TITLE
Potentially important feature and info request

### ISSUE THREAD

**Issue by SereneFrost72** [platky/PkmnCardsOnline-Frontend#18]
Hey again,

I e-mailed this to the pkmncardsonline@gmail.com address, but I wasn't sure you received it.

Once you implement the deck creation/saving/etc. functionality, it would be amazing for competitive players to be able to simulate opening hands/prize set-ups/first few card draws. I did this in Excel, but I think this would be an amazingly attractive and distinctive feature for your website. I can send my Excel file if you'd like, but I'd imagine this would be much easier when you can write an actual script/code for it instead of using Excel formulas.

Also, can you explain how the site determines the card prices? I have been using your site as a guide when I buy singles, so I want to understand exactly what sources are used and how the prices are calculated.

Thanks for all of your efforts!

**Comment by platky** [comment]
I did get your message I just forgot to respond. I have seen your handy work on reddit and think its a great idea to add in and like you said it would actually be very easy through code once its set up. I could even throw in some statistics and probabilities for various things in there.


As for price information currently. Depending on the rarity of the card there are some minor variances. Right now it polls listings on Ebay (and sometimes Amazon), grabs pricing info from a few of the major sites such as trollandtoad and coolstuffinc and averages them out. I have found that alone generates prices that are overall pretty accurate but a little high so I also threw in a percentage scale back, which I think is around 5% currently. This is how it works for 99.9% of the cards, there are about 10 that are hardcoded in currently because my general algorithms wasnt scraping info properly. It still needs a little fine tuning I think but as a method of price estimation it has worked very well for me so far. I will also throw up some information about this on the website since I am sure many others will be curious about this.

Thanks for your questions and feedback, your help has been very beneficial!
